src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
+}
+
+function checkHealth() {
+  Animal.alive = Animal.alive.filter(animal => animal.health > 0);
 }
```

Comment:
The function checkHealth() is a global function which is supposed to operate on data of Animal class. It would be better to include it as a static method inside the Animal class.